src/main/java/view/view_events/EventType.java

```diff
@@ -1,11 +1,10 @@
 package view.view_events;
 
 public enum EventType {
```

Comment:
Since we get rid of checking for event types when sending the event in `ViewManager`, maintaining `EventType` is no longer necessary as `IComponent` classes can simply use `instanceof` to determine event types. Remove this `enum` and its all usages.

Reply:
Just forget...